docs/publishing.md

```diff
@@ -10,23 +10,27 @@ sidebar_label: Publishing Packages
 
 1. Make changes and commit them to origin/branch
 2. Ensure your `git status` is porcelian
-3. On your local machine `npm run version`
+3. On your local machine
+   `npm version [<newversion> | major | minor | patch | premajor | preminor | prepatch | prerelease [--preid=<prerelease-id>] | from-git]`
 4. Commit the automatically generated changes
 5. Raise a PR to origin/master
 6. The CI pipeline will then publish the changes for you
 
 ## What registry are we using?
 
-All of our packages are opensource and can be found under the @amidostacks organisation on [npm](https://www.npmjs.com/settings/amidostacks/packages).
+All of our packages are opensource and can be found under the @amidostacks
+organisation on [npm](https://www.npmjs.com/settings/amidostacks/packages).
```

Comment:
Wrong URL - this is authenticated. Try: https://www.npmjs.com/~amidostacker

Reply:
I did not change anything here